pkgs/development/libraries/aws-sdk-cpp/default.nix

```diff
@@ -114,6 +115,10 @@ stdenv.mkDerivation rec {
   # Builds in 2+h with 2 cores, and ~10m with a big-parallel builder.
   requiredSystemFeatures = [ "big-parallel" ];
 
+  passthru.tests = {
+    inherit nix;
+  };
+
```

Comment:
I'd consider pulling whole `nix.tests` as well, as `nix` is quite important for us.  Though that will probably fail to evaluate until PR #188977

Reply:
good idea, added